README.md

```diff
@@ -51,6 +51,42 @@ This document provides the information needed to create a DASH packager that is
 
   See https://github.com/google/edash-packager/blob/master/CONTRIBUTING.md for details.
 
+# Using docker for testing/development #
+
+1. Requirements:
+
+   1. Clone this repo.
```

Comment:
Will it work without cloning the repo? It seems a waste since the repo is not really used except the Dockerfile. 

Does docker work by providing github url, something like: 

docker build  github.com/google/edash-packager  

or maybe

docker build https://github.com/leandromoreira/edash-packager/blob/master/Dockerfile